backend/urls.py

```diff
@@ -41,7 +41,7 @@
     path('', views.get_home_template, name = 'home'),
 
     # For file storage
-    re_path(r'^files/', include('db_file_storage.urls')),
+    re_path(r'^files/', include('db_file_storage.urls'), {'extra_headers': {'Cache-Control': 'public, max-age=31536000'}}),
```

Comment:
Can you translate 31536000 into hours/days?

Reply:
It's basically a year, what's generally suggested as a "constant" on various places

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#caching_static_assets_with_cache_busting

https://ashton.codes/set-cache-control-max-age-1-year/

If new file is uploaded then the name changes so it would be refreshed. This really can be cached forever.

Reply:
Got it. Good to have those info in the comment.

Reply:
Did you find a doc for how to set `extra_headers` with this library? Can you link to that if so? Here or in the source code, I'm find either way.

You could also do something like `(365 * 24 * 60 * 60)` for increased readability on the age.

Reply:
It was on an [issue filed against this library.](https://github.com/victor-o-silva/db_file_storage/issues/103)